src/modules/NtpClient.rb

```diff
@@ -697,17 +697,23 @@ def Export
       # restrict_map is a map with the key ip,ipv4-tag or ipv6-tag.
       # This will be converted into a list in order to use it in
       # autoyast XML file properly.
-      restricts = @restrict_map.collect do |target, values|
+      restricts = deep_copy(@restrict_map)
```

Comment:
I would try to not introduce more usage of `deep_copy` (is one of those leftovers from ycp -> ruby migration we would like to die).

What about?
```ruby
restrics = @restrict_map.collect do |target, values|
  # cfa_record not needed for export
  export_values = values.dup.delete("cfa_record")
  export_values["target"] = target
  export_values
end

peers = @ntp_records.map do |peer|
  # cfa_record not needed for export
   peer.dup.delete("cfa_record")
end
```
It's the same but without "ruby aliens"